src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -1,371 +1,184 @@
-                            ">
-                                Documentation
-                        </Button>
-                    </> }
-                    onNext={ this.onStep }
-                    onGoToStep={ this.onStep }
-                    steps={ steps }
-                    onClose={ this.onClose }
-                    onSave={ this.onSave }
-                    footer={ <ImageWizardFooter
-                        isValidUploadDestination={ isValidUploadDestination }
-                        isSaveInProgress={ this.state.isSaveInProgress }
-                        isValidSubscription={ this.state.isValidSubscription }
-                        error={ this.props.composesError } /> }
-                    isOpen />
-            </React.Fragment>
-        );
-    }
-}
-
-function mapStateToProps(state) {
-    return {
-        composesError: state.composes.error,
-        release: state.pendingCompose.release,
-        uploadDestinations: state.pendingCompose.uploadDestinations,
-        uploadAWS: state.pendingCompose.uploadAWS,
-        uploadAzure: state.pendingCompose.uploadAzure,
-        uploadGoogle: state.pendingCompose.uploadGoogle,
-        selectedPackages: state.pendingCompose.selectedPackages,
-        subscription: state.pendingCompose.subscription,
-        subscribeNow: state.pendingCompose.subscribeNow,
-    };
-}
-
-function mapDispatchToProps(dispatch) {
-    return {
-        composeUpdated: (compose) => dispatch(actions.composeUpdated(compose)),
-        composeStart: (composeRequest) => dispatch(actions.composeStart(composeRequest)),
-        addNotification: (not) => dispatch(addNotification(not)),
-    };
-}
-
-CreateImageWizard.propTypes = {
-    composesError: PropTypes.string,
-    composeUpdated: PropTypes.func,
-    composeStart: PropTypes.func,
-    addNotification: PropTypes.func,
-    history: PropTypes.object,
-    release: PropTypes.object,
-    uploadDestinations: PropTypes.object,
-    uploadAWS: PropTypes.object,
-    uploadAzure: PropTypes.object,
-    uploadGoogle: PropTypes.object,
-    subscription: PropTypes.object,
-    subscribeNow: PropTypes.bool,
-    selectedPackages: PropTypes.array,
+            ">
+                Documentation
+                    </Button></div>,
+                    // order in this array does not reflect order in wizard nav, this order is managed inside
+                    // of each step by `nextStep` property!
+                    fields: [
+                        imageOutput,
+                        awsTarget,
+                        googleCloudTarger,
+                        msAzureTarget,
+                        registration(user),
+                        packages,
+                        review,
+                    ]
+                }
+            ]
+        } } /> : <Spinner />;
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(withRouter(CreateImageWizard));
+export default CreateImage;
```

Comment:
Currently the `CreateImageWizard.js` file exports `CreateImage` and the `ImageCreator.js` file exports `CreateImageWizard`. Can you unify the naming so each file creates and exports a component with the same name as it?